CHANGELOG.md

```diff
@@ -16,10 +16,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   DatePicker: add an input to change the displayed year
 -   Message: add a `closeButtonProps` prop to add a close button in the message. Only available for `info` kind messages with a background.
+-   Popover: add a `focusTrapZoneElement` prop to specify the element in which the focus trap should be applied.
 
 ### Changed
 
 -   Dialog: fullscreen display on smaller viewport
+-   FocusTrap: Make the focus trap work on element with shadow root
```

Comment:
Don't need to mention that in the changelog because the lumx libs do not expose the focus trap utility.
We only list changes that affect the the outside world

```suggestion
```